.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     # Specify service dependencies here if necessary
     # CircleCI maintains a library of pre-built images
     # documented at https://circleci.com/docs/2.0/circleci-images/
-    - image: mysql:5.5
+    - image: mysql:8.0
```

Comment:
Is this version change intended by Young?

Reply:
I changed this since MySQL with the version under 8.0 does not support row_number() .